test/relations.test.js

```diff
@@ -4987,7 +4987,7 @@ describe('relations', function() {
       Category.findById(category.id, function(err, cat) {
         if (err) return done(err);
         var link = cat.items.at(0);
-        link.updateAttributes({notes: 'Updated notes...'}, function(err, link) {
+        cat.items.updateById(link.id, {notes: 'Updated notes...'}, function(err, link) {
```

Comment:
My understanding of embedded relations is rather limited. However, by looking at the test name, test code and surrounding tests, I think the very purpose of this test is to verify how `link.updateAttributes` behaves. Changing `link.updateAttributes` to `cat.items.updateById` is making the test pointless IMO.